runtime/kusama/src/lib.rs

```diff
@@ -77,7 +77,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("kusama"),
 	impl_name: create_runtime_str!("parity-kusama"),
 	authoring_version: 2,
-	spec_version: 1040,
+	spec_version: 1041,
```

Comment:
why is this required?

Reply:
This was part of #769 which I merged in, so @expenses may have more background.

Reply:
It might not be needed, CI have me an error before though.